exporter/signalfxexporter/hostmetadata/metadata.go

```diff
@@ -82,6 +82,8 @@ func (s *Syncer) syncOnResource(res pdata.Resource) {
 }
 
 func (s *Syncer) prepareMetadataUpdate(props map[string]string, hostID splunk.HostID) *metadata.MetadataUpdate {
+	s.logger.Info("Preparing to sync host properties to host dimension", zap.String("dimension_key", string(hostID.Key)),
```

Comment:
I wonder if it'd make more sense to put this after line 74 since this function is just returning a value but not doing any updating.

Reply:
Sounds good to me. I like that to keep the function concise.